feature-libs/cart/quick-order/styles/_quick-order-form.scss

```diff
@@ -1,12 +1,10 @@
 cx-quick-order-form {
-  .cx-quick-order-container {
-    align-items: center;
+  .quick-order-form-container {
     display: flex;
   }
 
   .quick-order-form-input {
-    align-items: center;
-    display: flex;
+    margin: 0 15px 0 0;
```

Comment:
I saw this was done to fix the failed pipeline https://github.com/SAP/spartacus/pull/13245/commits/4649677e4b7c43578b692694d70afbe6ba935d84

I'm assuming,  you don't want to set top left bottom to 0, but just make the margin-right to have 15px right?

You should use ` margin-inline-end: 15px` if you wanted to keep the margin-right only to be 15px.

P.S margin-right / left and padding-right/left 

are done like margin-inline-end/start and padding-inline-end/start to support `directionality`

Reply:
thank you :) didn't know that I should use it :)